README.md

```diff
@@ -141,19 +141,25 @@ There may also be API packages for experimental signals in the experimental dire
 All stable packages are released with the same version, and all experimental packages are released with the same version.
 The below table describes which versions of each set of packages are expected to work together.
 
-| API   | Stable Packages | Experimental Packages |
-| ----- | --------------- | --------------------- |
-| 1.3.x | 1.9.x           | 0.35.x                |
-| 1.3.x | 1.8.x           | 0.34.x                |
-| 1.2.x | 1.7.x           | 0.33.x                |
-| 1.2.x | 1.6.x           | 0.32.x                |
-| 1.1.x | 1.5.x           | 0.31.x                |
-| 1.1.x | 1.4.x           | 0.30.x                |
-| 1.1.x | 1.3.x           | 0.29.x                |
-| 1.1.x | 1.2.x           | 0.29.x                |
-| 1.1.x | 1.1.x           | 0.28.x                |
-| 1.0.x | 1.0.x           | 0.27.x                |
-| 1.0.x | 1.0.x           | 0.26.x                |
+| Stable Packages                                                     | Experimental Packages |
+|---------------------------------------------------------------------|-----------------------|
+| 1.15.x                                                              | 0.41.x                |
+| 1.14.x                                                              | 0.40.x                |
+| 1.13.x                                                              | 0.39.x                |
+| 1.12.x                                                              | 0.38.x                |
+| 1.11.x                                                              | 0.37.x                |
+| 1.10.x                                                              | 0.36.x                |
+| 1.9.x                                                               | 0.35.x                |
+| 1.8.x (this and following versions require API >=1.3.0 for metrics) | 0.34.x                |
+| 1.7.x                                                               | 0.33.x                |
+| 1.6.x                                                               | 0.32.x                |
+| 1.5.x                                                               | 0.31.x                |
+| 1.4.x                                                               | 0.30.x                |
+| 1.3.x                                                               | 0.29.x                |
+| 1.2.x                                                               | 0.29.x                |
+| 1.1.x                                                               | 0.28.x                |
+| 1.0.x                                                               | 0.27.x                |
+| 1.0.x (this and following versions require API >=1.0.0 for traces)  | 0.26.x                |
```

Comment:
I'm not sure if the word "following" versions is referring to the versions above this row -- as there are no versions below this line.

```suggestion
| 1.0.x (this and above versions require API >=1.0.0 for traces)  | 0.26.x                |
```

Reply:
yes that's the word I was looking for. [52de9f1](https://github.com/open-telemetry/opentelemetry-js/pull/4104/commits/52de9f16379fae5a3636e429ca594f940189f8de)